docs/errors/E0326.md

```diff
@@ -0,0 +1,20 @@
+# E0326: 'async export' is not allowed; write 'export async' instead
+
+Class methods can have modifiers such as `export`, `static`, `async`, and `*`. It is a
```

Comment:
Class methods can be marked `export`...? I'm confused by this wording. It seems wrong and unrelated to `async export function f() {}`.